packages/@n8n/config/src/configs/runners.config.ts

```diff
@@ -53,4 +60,12 @@ export class TaskRunnersConfig {
 	/** Should the output of deduplication be asserted for correctness */
 	@Env('N8N_RUNNERS_ASSERT_DEDUPLICATION_OUTPUT')
 	assertDeduplicationOutput: boolean = false;
+
+	/** How long (in minutes) until shutting down an idle runner. */
+	@Env('N8N_RUNNERS_IDLE_TIMEOUT')
+	idleTimeout: number = 5;
+
+	/** How often (in minutes) to check if a runner is idle. */
+	@Env('N8N_RUNNERS_IDLE_CHECKS_FREQUENCY')
+	idleChecksFrequency: number = 1;
```

Comment:
Do we need to make this configurable or could we somehow calculate it from the idle timeout?